api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -270,6 +270,16 @@ public void createAllOfUsBillingProject(String projectName) {
         });
   }
 
+  @Override
+  public void deleteBillingProject(String billingProject) {
```

Comment:
Does this behave the same for user-provided billing projects and free tier billing projects/accounts? Do we need to update spend totals before killing it?

Reply:
We will always own the whole lifecycle for billing *projects* - users will bring billing *accounts* in the future which we will assign to the already-existing projects.

Reply:
that makes more sense, thanks.